src/pages/Home.tsx

```diff
@@ -1,12 +1,51 @@
-import type { FunctionComponent } from "../common/types";
+import { useQuery } from "@tanstack/react-query";
+import type { FunctionComponent, Player } from "../common/types";
+import PlayerSearch from "../features/player-search/components";
+import { getPlayerSearch } from "../features/player-search/api";
+import { useEffect, useState } from "react";
+import { random } from "lodash";
+import { Box } from "@mui/material";
+import VirtualPlayerList from "../components/ui/player-list";
 
 export const Home = (): FunctionComponent => {
+  const [queryEnabled, setQueryEnabled] = useState<boolean>(false)
+  const [searchQuery, setSearchQuery] = useState({playerName: ''})
+  const [queryId, setQueryId] = useState<number>(random(1, 1000))
+  const [playerData, setPlayerData] = useState<Array<Player>>([])
+
+  // Search query is made from the page level, not the component level
+  const {data} = useQuery<Array<Player>>({
+    enabled: queryEnabled,
+    queryKey: ['player-search', queryId ,searchQuery],
+    queryFn: async () => getPlayerSearch(searchQuery.playerName),
+  });
+
+  const handleSubmit = (playerName: string): void => {
+    if (!playerName || playerName.length < 3) {
+      setPlayerData([]);
```

Comment:
Clear out the state of the data if we have less than the necessary characters for an API call.